README.md

```diff
@@ -1,58 +1,56 @@
 # Micromasters
 Portal for learners and course teams to access MITx Micromasters programs
 
-## Getting Started
+## Major Dependencies
+- pip, virtualenv, homebrew (OSX only)
+- Postgres >= 9.4
+  - OSX recommended install method: [Postgres.app](http://postgresapp.com/)
+- Python 3
+  - OSX recommended install method: Homebrew `brew install python3`
+- Docker (_optional, but strongly recommended_)
+  - OSX recommended install method: [Download from Docker website](https://docs.docker.com/mac/)
+- docker-compose (_if you're using Docker, this is required_)
+  - Recommended install: pip `pip install docker-compose`
 
-Although you can run Micromasters locally with a default sqlite database after
-installing the ``requirements.txt`` file, the recommended way is to 
-use [Docker](https://www.docker.io). Install Docker, then install 
-``docker-compose`` and run the app: 
+## (OSX only) Getting your machine Docker-ready
 
-    pip install docker-compose 
-    docker-compose up
+#### Install Python dependencies and create your docker container:
```

Comment:
The Docker container should include all setup of the Python dependencies. Since we're using it for Linux and OS X anyway, why not leave this information out?